src/files/BrsFile.ts

```diff
@@ -138,7 +138,23 @@ export class BrsFile {
         }
     }
 
-    public parser = new Parser();
+    public get parser() {
+        if (!this._parser) {
+            //remove the typedef file (if it exists)
+            this.hasTypedef = false;
```

Comment:
these need to be removed before calling `parse`, because `parse` will stop parsing if it detects a typedef file. It's a bit of a hack, but since it's all internal logic, it works fine.